eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -619,4 +619,23 @@ public function publish( $contentTypeId )
 
         $this->updateHandler->publishNewType( $toType, Type::STATUS_DEFINED );
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
+     */
+    public function getFieldMap()
+    {
+        $fieldMap = [];
```

Comment:
One practical note about that.

I haven't explored the feature yet, but phpstorm supports warnings about PHP features, depending on the project's version. If we have like a project per major version, each with the minimal php version configured, we should be able to benefit from it.

It also highlights that backports should trigger travis tests :-)